README.md

```diff
@@ -94,6 +94,19 @@ Fluid Dialogue is used through [Unity's Package Manager](https://docs.unity3d.co
 
 The following APIs are available for customizing your dialogue experience.
 
+### Initializing a Dialogue
+
+You can initialize a dialogue using the .Play method, as show in the following script : 
+https://github.com/ashblue/fluid-dialogue/blob/develop/Assets/Examples/BasicConversation/Scripts/ExampleDialoguePlayback.cs#L56
```

Comment:
Line numbers change so let's remove that, but linking to the file is a great idea.

```suggestion
https://github.com/ashblue/fluid-dialogue/blob/develop/Assets/Examples/BasicConversation/Scripts/ExampleDialoguePlayback.cs
```